File: Iniciante/URI 1963.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var line = lines.shift().split(' ');
var A = parseFloat(line[0]);
var B = parseFloat(line[1]);

var aumento = (B * 100.0 / A) - 100.0;
console.log(aumento.toFixed(2) + '%');

File: Iniciante/URI 1963.clj
(require '[clojure.string :as str])

(def line (str/split (read-line) #" "))
(def A (Double/parseDouble (get line 0)))
(def B (Double/parseDouble (get line 1)))

(def aumento (- (/ (* B 100.0) A) 100.0))

(printf "%.2f%%\n" aumento)